infra/database/type/opengauss/src/main/java/org/apache/shardingsphere/infra/database/opengauss/metadata/data/loader/OpenGaussMetaDataLoader.java

```diff
@@ -70,7 +72,10 @@ public final class OpenGaussMetaDataLoader implements DialectMetaDataLoader {
     @Override
     public Collection<SchemaMetaData> load(final MetaDataLoaderMaterial material) throws SQLException {
         try (Connection connection = material.getDataSource().getConnection()) {
-            Collection<String> schemaNames = SchemaMetaDataLoader.loadSchemaNames(connection, TypedSPILoader.getService(DatabaseType.class, "openGauss"));
+            DatabaseType databaseType = TypedSPILoader.getService(DatabaseType.class, "openGauss");
```

Comment:
What does this code mean?

Reply:
The databaseType will be used as a parameter in the two code sections below, so it is computed beforehand for use.